src/Winium.Desktop.Driver/CommandExecutors/NewSessionExecutor.cs

```diff
@@ -32,6 +31,17 @@ protected override string DoImpl()
             // Gives sometime to load visuals (needed only in case of slow emulation)
             Thread.Sleep(this.Automator.ActualCapabilities.LaunchDelay);
 
+            // Update running application process in Application class if it's exited base on input process name
+            if (this.Automator.Application.HasExited())
```

Comment:
@zelonght change suggested code into 
if (this.Automator.Application.HasExited() && this.ExecutedCommand.Parameters["desiredCapabilities"]["processname"] != null) to avoid error in compile.